src/hooks/useScreenReader.test.ts

```
@@ -0,0 +1,69 @@
+
+  it('should detect high contrast preference', () => {
+    mockMatchMedia.mockReturnValue({ matches: true });
+    expect(prefersHighContrast()).toBe(true);
+
+    mockMatchMedia.mockReturnValue({ matches: false });
+    expect(prefersHighContrast()).toBe(false);
+  });
+
+  it('should handle window not defined', () => {
+    const originalWindow = global.window;
+    delete (global as Record<string, unknown>).window;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)